.github/workflows/updatecli.yml

```diff
@@ -14,16 +14,15 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: elastic/apm-pipeline-library/.github/actions/updatecli@current
-        env:
-          BRANCH_NAME: ${{ github.ref_name }}
         with:
           vaultUrl: ${{ secrets.VAULT_ADDR }}
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
-          pipeline: ./.ci/update-specs.yml
+          pipeline: .ci/updatecli.d
       - if: failure()
         uses: elastic/apm-pipeline-library/.github/actions/notify-build-status@current
         with:
+          status: ${{ steps.check.outputs.status }}
```

Comment:
leftover?

```suggestion
```

Reply:
Thank you for your eagle eyes. 🦅

Reply:
https://github.com/elastic/apm-agent-nodejs/pull/3197/commits/2a3c66a84b413aee36c6b24a3fe1729eea198c14